draft-ietf-quic-http.md

```diff
@@ -127,27 +127,36 @@ port.
 
 This document defines the "quic" parameter for Alt-Svc, which MAY be used to
 provide version-negotiation hints to HTTP/QUIC clients. QUIC versions are
-four-octet sequences with no additional constraints on format. Syntax:
+four-octet sequences with no additional constraints on format.  Leading zeros
+SHOULD be omitted for brevity.
+
+Syntax:
 
 ~~~ abnf
-quic = version-number
+quic = DQUOTE version-number [ "," version-number ] * DQUOTE
 version-number = 1*8HEXDIG; hex-encoded QUIC version
 ~~~
 
-Leading zeros SHOULD be omitted for brevity.  When multiple versions are
-supported, the "quic" parameter MAY be repeated multiple times in a single
-Alt-Svc entry.  For example, if a server supported both version 0x00000001 and
-the version rendered in ASCII as "Q034", it could specify the following header:
+Where multiple versions are listed, the order of the values reflects the
+server's preference (with the first value being the most preferred version).
+Reserved versions MAY be listed, but unreserved versions which are not supported
+by the alternative SHOULD NOT be present in the list. Origins MAY omit supported
+versions for any reason.
+
+Clients MUST ignore any included versions which they do not support.  The "quic"
+parameter MUST NOT occur more than once; clients SHOULD process only the first
+occurrence.
+
+For example, if a server supported both version 0x00000001 and the version
+rendered in ASCII as "Q034", it could specify the following header:
 
 ~~~ example
-Alt-Svc: hq=":49288";quic=1;quic=51303334
+Alt-Svc: hq=":49288";quic="1,1abadaba,51303334,0"
 ~~~
 
-Where multiple versions are listed, the order of the values reflects the
-server's preference (with the first value being the most preferred version).
-Origins SHOULD list only versions which are supported by the alternative, but
-MAY omit supported versions for any reason.
-
+A client acting on this header would drop the reserved versions (because it does
```

Comment:
For clarity, can you add a note first about 1abadaba and 0 being reserved versions, with a ref to Section 4 of the transport draft?